nepalingo-web/src/hooks/useNewari.tsx

```diff
@@ -1,37 +1,39 @@
-import useSWR from 'swr'
-import { DictionaryProps, DictionaryResponse } from './useDictionary'
+import useSWR from "swr";
+import { DictionaryProps, DictionaryResponse } from "./useDictionary";
 
+const fetcher = (url: string) =>
+  fetch(import.meta.env.VITE_NEPALBHASA_API_URL + url, {}).then((r) =>
+    r.json()
+  );
 
-const fetcher = (url: string) => fetch(import.meta.env.VITE_NEPALBHASA_API_URL + url, {
-}).then(r => r.json())
+const useNewari = (props: Omit<DictionaryProps, "language">) => {
+  let { data, error, isLoading } = useSWR(
```

Comment:
looks like your branch might be out of date - can you merge in main into this branch (or rebase) ?